### DISCUSSION TITLE
Use current as default in log other prompt

### DISCUSSION THREAD

**Discussion by svenssonaxel** [magit/magit#5045] [Ideas]
To reproduce:
- Press `l o` to produce a log with custom starting points. You get a prompt `rev,s:`
- Enter `branch1,branch2`.
- When the log shows, press `l o` again to change the branch selection.

Desired:
- You get a prompt `rev,s:` with the text `branch1,branch2` pre-filled

Actual:
- You get a prompt `rev,s:` with nothing pre-filled

**Comment by svenssonaxel** [comment]
Only vaguely related to #5042, as this is about choosing a set of revisions, not a single one, but it could be worth mentioning.

**Comment by tarsius** [comment]
Previous values are available through the minibuffer history, `M-p`/`M-n` during completion. Why isn't that enough?

**Comment by svenssonaxel** [comment]
TIL, that's great. Thanks!

**Reply by tarsius** [reply]
:smiley_cat: Works for nearly everything that does completion!
